examples/component-versioning/component-mutable.md

```diff
@@ -95,7 +106,7 @@ spec:
   - env:
     - name: TEST_ENV
       value: test
-    image: wordpress:4.6.1-apache
+    image: wordpress:php7.2
```

Comment:
Why make this tag change?

Reply:
the original `sample_application_config` example modified the image parameter, this is the result of the parameter render: 
https://github.com/crossplane/oam-kubernetes-runtime/blob/55cd9dbe93cfeba4fe03dfd8c81c34a4f367d2bc/examples/containerized-workload/sample_application_config.yaml#L8-L12